README.md

```diff
@@ -9,7 +9,7 @@ Thank you for your [![Stars](https://img.shields.io/github/stars/bcgsc/LINKS.svg
 # LINKS
 
 ## Long Interval Nucleotide K-mer Scaffolder
-## LINKS v1.8.7 Rene L. Warren, 2014-2021
+## LINKS v2.0 Rene L. Warren, 2014-2021
```

Comment:
```suggestion
## LINKS v2.0.0 Rene L. Warren, 2014-2021
```
Throughout, should be v2.0.0 to have proper semantic versioning

Reply:
minor (but important): murathan, let's please add Yaman and your name to developers